tests/utils/pod/pod.go

```diff
@@ -57,3 +57,19 @@ func RedefineWithLivenessProbe(pod *corev1.Pod) *corev1.Pod {
 
 	return pod
 }
+
+func RedefineWithVolume(pod *corev1.Pod, name string) *corev1.Pod {
```

Comment:
Same here, I don't think you need to return the pointer.

Reply:
correct, thanks.
Will fix and upload a new version, I will also implement this across other functions.

Reply:
Agreed, it can be addressed later just pointing it out.

Reply:
I agree and i think it's better to do it in a different PR, 
I will start working on one.